File: 06. Perulangan/B. For/b_for.c
#include <stdio.h>
int main(){
  int N, i, B, res;
  scanf("%d", &N);
  res = 0;
  for (i = 0; i < N; i++){
    scanf("%d", &B);
    res += B;
  }
  printf("%d\n", res);
  return 0;
}

File: 06. Perulangan/B. For/b_for.go
package main

import (
  "fmt"
)

func main() {
  var N, B, res int
  fmt.Scanln(&N)
  res = 0
  for i := 0; i < N; i++ {
    fmt.Scanf("%d", &B)
    res = res + B
  }
  fmt.Println(res)
}
